### PR TITLE
Add a makeshift docstrings

### PR DESCRIPTION
**Pull request by Paalon**
It's probably better than nothing. I'm not a native English speaker, thus there might be unnatural sentences.

### CODE REVIEW

**Review by kescobo — APPROVED** [review]
Definitely better than nothing. All the English is clear to me.

xref #136

**Comment by GunnarFarneback** [comment]
I'm not a native speaker either, but I'll add comments to formulations which look strange to me.

File: src/YAML.jl [hunk 1]
```diff
@@ -1,3 +1,22 @@
+"""
+    YAML
+
+A package to treat YAML.
```
Comment (GunnarFarneback) [inline src/YAML.jl:4]: "treat" -> "read and write"

File: src/YAML.jl [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+https://github.com/JuliaData/YAML.jl
+
+Reading:
+
+* `YAML.load` parses the first YAML document of a YAML file as a Julia object.
+* `YAML.load_all` parses the all YAML documents of a YAML file.
```
Comment (GunnarFarneback) [inline src/YAML.jl:10]: "the all" -> "all"

File: src/YAML.jl [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+* `YAML.load_file` is same with `YAML.load` except it reads from a file.
```
Comment (GunnarFarneback) [inline src/YAML.jl:11]: "same with" -> "same as"

**Comment by kescobo** [comment]
Derp - Agree with all of those, apparently I didn't read very closely :facepalm:

**Pull request merged**